ATS/db/schema.rb

```diff
@@ -179,6 +179,8 @@
     t.string   "uid"
     t.string   "email"
     t.string   "name"
+    t.string   "first_name"
```

Comment:
@sah0017 @JavanR  This schema looks way out of date. There were no changes to the migration files in this pull request so I'm not sure why this schema looks like this. It might be a good idea to do a db:reseed or reset your schema and regenerate it and then commit again into this pull request with that updated schema.